inst/help/MixedModelsBGLMM.md

```diff
@@ -0,0 +1,149 @@
+Bayesian Generalized Linear Mixed Models
+===
+
+Bayesian Generalized Linear Mixed Models allow you to model a linear relationship between one or more explanatory variable(s) and a continuous dependent variable in cases where the observations are not independent, but clustered given one or several random effects grouping factors (e.g., repeated measures across participants or items, children within schools). They are generalization of Bayesian Linear Mixed Models and allow to model response variables that are not continous using a different likelihoods and link functions.
+
+### Assumptions
+- Linearity and additivity: The response variable is linearly related to all predictors and the effects of the predictors are additive on the linear scale.
```

Comment:
minor comment: the response variable does not have to be linearly related, there is a link function involved too!